Polyhedron/demo/Polyhedron/Plugins/PCA/Clipping_box_plugin.cpp

```diff
@@ -88,6 +88,12 @@ dock_widget->show();
     item = new Scene_edit_box_item(scene);
   connect(item, SIGNAL(destroyed()),
           this, SLOT(enableAction()));
+  connect(item, &Scene_edit_box_item::aboutToBeDestroyed,
+          [this]()
+  {
+    clip(false);
+    dock_widget->pushButton->setChecked(false);
+  });
```

Comment:
When you use that new version of `connect`, use anyway the version with four arguments:
```c++
connect(item, &Scene_edit_box_item::aboutToBeDestroyed,
        this, [this]()
{
  clip(false);
  dock_widget->pushButton->setChecked(false);
});
```

The extra argument (the third, here `this`), ensures that if the targeted
object of the connection is destroyed, then the connection is removed just
before. Otherwise the application could run a lambda expression with
dandling references/pointers to objects that were destroyed.

Well, in this case, that was safe, because it is unlikely, because `this`
is a plugin of the application, and we do not have any way to unload and
destroy a plugin before the exit of the application.